README.md

```diff
@@ -101,9 +107,10 @@ If no runtime setting is provided at all, the plugin will fail.
 
 Similarly, you can set the `source` location of each function in case you keep the code in separate folders.
 
-There are two ways to set environment variables when deploying cloud functions:
+There are three ways to set environment variables when deploying cloud functions:
 - from a secret
 - putting the value directly into the drone.yml file
+- Google Secret Manager
```

Comment:
Question: is it clear in what format Secret  Manager secrets are being passed? Is there a link to some GCP documentation that could help?